operator/v1/tests/ingresscontrollers.operator.openshift.io/SetEIPForNLBIngressController.yaml

```diff
@@ -0,0 +1,333 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "IngressController"
+crdName: ingresscontrollers.operator.openshift.io
+featureGate: SetEIPForNLBIngressController
+tests:
+  onCreate:
+    - name: Should be able to create a minimal IngressController
+      initial: |
+        apiVersion: operator.openshift.io/v1
+        kind: IngressController
+        spec: {} # No spec is required for a IngressController
```

Comment:
I'm surprised `metadata` isn't required.

Reply:
That's an artifact of the test suite here which will inject metadata for name and namespace as required

Reply:
hmm I am also not sure on this part.

Reply:
No, the test suite handles this for you